Notice part

AlexAX135 opened the pull request.

AlexAX135 committed:
Update MainDefinitions.ix

Definition for note order

AlexAX135 committed:
Add files via upload

AlexAX135 committed:
Update ServiceFrame-2017.ix

Added notices & NoticeAssignment

AlexAX135 committed:
Update MainDefinitions.ix

AlexAX135 committed:
Update NoticesAssignment.ix

skinkie commented on CompositeFrame/ServiceFrame-2017.ix:
```diff
@@ -31,4 +31,8 @@ xmlelement ServiceFrame {
+	
+	&include ServiceFrame/notices.ix
+	
+	&include ServiceFrame/NoticesAssignment.ix
```
Could you rename NoticesAssignment into noticeAssignments.ix? If it is a hassle, I can do it for you.

AlexAX135 committed:
Rename NoticesAssignment.ix to noticesAssignment.ix

AlexAX135 committed:
Update ServiceFrame-2017.ix

skinkie commented on CompositeFrame/ServiceFrame-2017.ix:
```diff
@@ -34,5 +34,5 @@ xmlelement ServiceFrame {
 	&include ServiceFrame/notices.ix
 	
-	&include ServiceFrame/NoticesAssignment.ix
+	&include ServiceFrame/noticesAssignment.ix
```
Het moet worden noticeAssigments, gelijk aan de XML tag waarmee het begint.

AlexAX135 committed:
Update ServiceFrame-2017.ix

AlexAX135 committed:
Rename noticesAssignment.ix to noticesAssignments.ix

AlexAX135 committed:
Update noticesAssignments.ix

id shoukd be ref

AlexAX135 committed:
Update Configuratie.ix

fix for multiple value errror in Hastus2020. StrLocateStr gives an index for eacht of the 4 options.
